Formula/qshell.rb

```diff
@@ -0,0 +1,14 @@
+class Qshell < Formula
+  desc "Qiniu Cloud Resource Management Tool "
+  homepage "https://github.com/qiniu/qshell"
+  url "https://dn-devtools.qbox.me/qshell-2.1.3.tar.gz"
```

Comment:
The [same docs](https://docs.brew.sh/Formula-Cookbook.html) as other formulae work here too, but you’ll have to host the formula in [your own tap](https://docs.brew.sh/How-to-Create-and-Maintain-a-Tap.html) as Homebrew disallow pre-built binaries in the official taps.

In your case the only thing you need to change (beside the syntax issue in the `test do` block) is the `install` method:

```ruby
def install
  bin.install "qshell"
end
```